db/migrate/20190227204757_recreate_delayed_jobs.rb

```diff
@@ -0,0 +1,22 @@
+class RecreateDelayedJobs < ActiveRecord::Migration
```

Comment:
Is there a schema migration that dropped this table? And is the table still in use from twillio?

Reply:
This should be part of a future investigation